main.py

```
@@ -0,0 +1,55 @@
+import time, re
+from typing import Annotated
+from fastapi import FastAPI, Form, UploadFile, File, Header, HTTPException
+from pydantic import EmailStr
+
+api = FastAPI()
+#----- 데모용 DB (현재는 임시 리스트) ----------------------------
+demo_db = []
+#----- 비밀번호 규칙 -------------------------------------------
+password_pattern = "^(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#$%^&*+=-]).{5,20}$"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "password_pattern"

(S105)